setuptools/config/_apply_pyprojecttoml.py

```diff
@@ -275,7 +275,7 @@ def _valid_command_options(cmdclass: Mapping = EMPTY) -> dict[str, set[str]]:
     return valid_options
 
 
-def _load_ep(ep: metadata.EntryPoint) -> tuple[str, type] | None:
+def _load_ep(ep: metadata.EntryPoint) -> tuple[str, ModuleType] | None:
```

Comment:
In general, `_load_ep` could return `tuple[str, Any]` actually, because they can point to any attribute inside of any module, right?

But in this particular file `_load_ep` will always load a class (which should be a subclass of `Command` or at least mimic the same protocol).

Reply:
Ah, I see how `importlib_metadata.EntryPoint.load`'s `functools.reduce` led me astray
`importlib.metadata.EntryPoint.load` is correctly typed as `Any` in typeshed, but `importlib_metadata.EntryPoint.load` kept the `ModuleType` inference